src/detectors/BEMC/CalorimeterHit_factory_EcalBarrelScFiRecHits.h

```diff
@@ -32,7 +32,7 @@ class CalorimeterHit_factory_EcalBarrelScFiRecHits : public eicrecon::JFactoryPo
         m_resolutionTDC=10 * dd4hep::picosecond;//{this, "resolutionTDC", 10 * ps};
 
         // zero suppression values
-        m_thresholdFactor=5.0;// from ATHENA's reconstruction.py
+        m_thresholdFactor=36.0488;// from ATHENA's reconstruction.py
```

Comment:
For future reference, we probably don't need this many significant figures. No need to change it here though.

Reply:
Okay.